setup.py

```diff
@@ -58,6 +58,7 @@ def run(self):
         'Werkzeug >= 0.11',
     ),
     extras_require={
+        'docs': ('sphinx', 'pallets-sphinx-themes'),
```

Comment:
Does ReadTheDocs allow you to specify extras for building? Last I checked, you could only specify a txt file, but maybe that's changed.

Reply:
i have no idea, i just took this from @r1b :D